config/development.json

```diff
@@ -33,6 +33,7 @@
 		"desktop-promo": true,
 		"devdocs": true,
 		"devdocs/redirect-loggedout-homepage": true,
+		"guidestours": true,
```

Comment:
Set to `false` and use `development.local.json` if we're merging this one first.